README.rst

```diff
@@ -156,6 +156,14 @@ there is no tight coupling to the ``blas-src`` version, so version selection is
 0.13        0.2.0        0.6.0
 =========== ============ ================ ==============
 
+------------
+BLAS on MSRV
+------------
+
+Although ``ndarray`` currently maintains an MSRV of 1.64.0, this is separate from the MSRV (either stated or real) of the various BLAS providers.
+As of the time of writing, ``openblas`` currently supports MSRV of 1.67.0.
+So, while ``ndarray`` and ``openblas-src`` are compatible, they can only work together with toolchains 1.67.0 or above.
```

Comment:
I wonder if ndarray can just use an older blas-src and older openblas-src for earlier Rust. I think that's the case, but I'm not sure we need to use it in CI. Thanks for fixing this

Reply:
No problem - I'll try seeing if there's an older openblas-src that we can use and add it to the documentation. But this all was the motivation for #1455, since I was wondering whether we want to bump to 1.67 (in order to make it easier for users to use BLAS)